packages/fxa-payments-server/src/lib/test-utils.tsx

```diff
@@ -182,6 +182,18 @@ export const defaultAppContextValue = (): AppContextType => ({
   accessToken: 'at_12345',
   config,
   queryParams: {},
+  matchMedia: jest.fn().mockImplementation(query => {
+    return {
+      matches: false,
+      media: query,
```

Comment:
Does this mock need to have all these properties? Looks like ` jest.fn(() => { return { matches: false } })` is what's used everywhere else

Reply:
These are necessary to prevent failures in typescript checks

Reply:
Yeah, seems like that could be fixed by making the type simpler from the wrapper function in `src/index.tsx`